docs/glossary.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This is a great list! Might be good to add "Multi-app", and "AR-in-VR" as well.

Reply:
Thanks! I think we'll add those in another PR after thinking about a good description for them :)